### PR TITLE
[8.0] migrate account_cutoff_prepaid and account_accrual_base

### PR DESCRIPTION
**Pull request by sbidoul**
Most work was done by @alexis-via on launchpad. I just extracted the 8.0 part and brought it here.

Then we improved a bit with:
- extensibility hooks
- a multi-company record rule
- some tests

### CODE REVIEW

File: account_cutoff_prepaid/__openerp__.py [hunk 1]
```diff
@@ -60,7 +60,7 @@
         'images/prepaid_revenue_journal_entry.jpg',
         'images/prepaid_revenue_done.jpg',
     ],
-    'installable': False,
+    'installable': True,
     'active': False,
     'application': True,
 }
```
Comment (lmignon) [inline account_cutoff_prepaid/__openerp__.py:66]: the module description should be extracted in a README.rst file

Comment (lmignon) [inline account_cutoff_prepaid/__openerp__.py:66]: the licence key is missing

**Comment by lmignon** [comment]
Appart from the two comments in the code :+1:

**Comment by yvaucher** [comment]
@sbidoul I have updated 8.0 by merging changes from 7.0 sorry for the inconvenience. It needs a rebase.

**Comment by sbidoul** [comment]
@yvaucher no inconvenience at all. I rebased.

**Comment by sbidoul** [comment]
I updated the PR description to more accurately reflect the work that has been done here.

**Comment by fclementic2c** [comment]
I did a quick tests with just one invoice and it works fine
Ok for me
:+1:

**Comment by yvaucher** [comment]
:+1: 
It could have been a bit better using the 8.0 API for onchanges and tests, anyway it's fine for the porting and I understand refactoring is out of scope.
Thanks for the work done

Note: there is still @lmignon comment to address about file `account_cutoff_prepaid/__openerp__.py`

**Comment by sbidoul** [comment]
We have been using account_cutoff_prepaid on several production systems since a few months now. Accountants are very happy with it.

**Comment by yvaucher** [comment]
@sbidoul If you fix the manifest file with license key and move description in README.rst, I merge it

**Comment by gurneyalex** [comment]
@sbidoul @yvaucher: I took care of the requested changes and merged.

**Comment by sbidoul** [comment]
Thanks @gurneyalex ! It's good to see this project gathering interest.

**Pull request merged**